core/server/worker/src/main/java/alluxio/worker/grpc/UfsFallbackBlockWriteHandler.java

```diff
@@ -84,7 +84,7 @@
   @Override
   protected BlockWriteRequestContext createRequestContext(alluxio.grpc.WriteRequest msg)
       throws Exception {
-    BlockWriteRequestContext context = new BlockWriteRequestContext(msg, FILE_BUFFER_SIZE);
+    BlockWriteRequestContext context = mBlockWriteHandler.createRequestContext(msg);
```

Comment:
In `BlockWriteHandler#createRequestContext()` it looks like it calls `mWorker.createBlockRemote`, but in this method it can also call `mWorker.createBlockRemote`. Will that be a problem?

Instead, should we just initialize the counters correctly in this method?

Reply:
Done